typescript/cli/src/commands/validator.ts

```diff
@@ -49,3 +64,60 @@ const addressCommand: CommandModuleWithContext<{
     process.exit(0);
   },
 };
+
+const preFlightCheckCommand: CommandModuleWithContext<{
+  chain: string;
+  validators: string;
+}> = {
+  command: 'preflightCheck',
```

Comment:
Recommend renaming this to simply `check` users need not type a long (possibly case senstitive, can't remember) command name.